extensions/cornerstone/src/utils/getActiveViewportEnabledElement.ts

```diff
@@ -2,8 +2,15 @@ import { getEnabledElement } from '@cornerstonejs/core';
 import { IEnabledElement } from '@cornerstonejs/core/types';
 
 import { getEnabledElement as OHIFgetEnabledElement } from '../state';
+import { ViewportGridService } from '@ohif/core';
+
+export default function getActiveViewportEnabledElement(viewportGridService: ViewportGridService, viewportId: string): IEnabledElement {
+  if (viewportId) {
+    const { element } = OHIFgetEnabledElement(viewportId) || {};
+    const enabledElement = getEnabledElement(element);
+    return enabledElement;
+  }
 
```

Comment:
I am not a fan of this either, for the same reason: the function name doesn't really match what it does. If `viewportId` is known, a simple `cornerstone.getEnabledElementByViewportId(viewportId)` does the job.
we don't need to edit this function